PYTHON_CDM_Conversion_code/hourly_qff_to_cdm_lite_v1.py

```diff
@@ -156,6 +157,13 @@ def main(station="", subset="", run_all=False, clobber=False):
                   "station_type","primary_station_id","station_name","quality_flag"
                   ,"data_policy_licence","source_id","secondary_id"]]
 
+        # set report type to 4 for ICAO or 0 for all other hourly stations
+        dft["report_type"]=df["temperature_Source_Station_ID"]
+        dft['report_type'] = dft['report_type'].fillna("NULL")
+        dft["report_type"]=df["report_type"].str[:4].astype('str')
+        dft['report_type'] = np.where(dft['report_type'].isin(["ICAO"]), dft['report_type'],"0")
```

Comment:
Neat - I didn't realise you could do that with `np.where` :)

Reply:
ok great thanks. So remind me what I need to do now ? commit the chnages?